Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -106,6 +115,15 @@
             "helpMarkDown": "Specify a name for the output variable that is generated for the URL of the Azure App Service. The variable can be consumed in subsequent tasks."
         },
         {
+            "name": "TakeAppOfflineFlag",
```

Comment:
Any reason this was moved up. Even thought this option is common for both Kudu and webdeploy, this was kept below so that it comes along with webdeploy params as this is a well known webdeploy option.